presto-native-execution/Makefile

```diff
@@ -107,22 +120,19 @@ tidy-fix: cmake			#: Fix clang-tidy issues in the master branch
 tidy-check: cmake		#: Check clang-tidy issues in the master branch
 	pushd "$$(pwd)/.." && presto-native-execution/velox/scripts/check.py tidy master && popd
 
-centos-container:			#: Build the linux container for CircleCi
-	$(MAKE) linux-container CONTAINER_NAME=centos
-
-linux-container:
```

Comment:
Would you update presto-native-execution/README.md since it has references to these deleted targets?

Reply:
Good point! done!